toolbox/siesta/barriers/BarriersBase.py

```diff
@@ -136,14 +150,15 @@ def setup(self):
         if self.neb_scheme is not None:
             if self.neb_scheme not in neb_schemes_available:
                 print("NOT IMPLEMENTED")
-                #sys.exit()
+                sys.exit()
             else:
                 print("NEB image scheme is: {}".format(self.neb_scheme))
                 #return self.neb_scheme
         if self.ghost == True:
```

Comment:
Again, proper documentation might clear lots of my confusion ;)
Is `ghost` really needed? Shouldn't it me trivially handled by sisl?

Reply:
Didn't expect that u went through pull so fast i just push it to be safe and work on it later. Jjaajajjaja but okay ill document the the base class tmrw then if u show me for the manual barrier then i could follow... Sorry 😅

Yes we need the ghost need ...cz in vacancy we put two ghost on each point ... So in both initial and final there will be a ghost on top of specie ... basically We first generate the path without any ghost later we put back the ghost in initial & final position...
 
Also i found out sisl have now Atomghost .. i have to update the coroseponding parts ..